File: Java Script/Java Script入門/problem_004.js
// HTMLを表示する
process.stdin.resume();
process.stdin.setEncoding('utf8');
process.stdout.write("<h1>hello world</h1>\n");   //改行しない場合は、console.logではなくprocess.stdout.write（スタンダードアウト ライト）関数
process.stdout.write("<p>世界の皆さん、\n");      // \nは改行コードで\nで折り返すようになる
process.stdout.write("<b>こんにちは</b></P>");

File: java/Java入門3/problem_004.java
// whileによるループ処理2

public class Main {
    public static void main(String[] args) {
        int i = 3;    // カウンタ変数の初期化
        while (i <= 10) {
            System.out.println("hello world " + i);    // 繰り返し処理
            i = i + 2;    // カウンタ変数の更新　+2ずつ上がる　カウンタ変数がないと無限ループになりエラーになってしまうので、カウンタ変数は必須
        }
        System.out.println ("last " + i);
    }
}